backend/src/tx.rs

```diff
@@ -0,0 +1,136 @@
+use subxt::ext::codec::Encode;
+use subxt::ext::sp_runtime::traits::{IdentifyAccount, Verify};
+use subxt::ext::{sp_core, sp_runtime};
+use subxt::{config::polkadot::PolkadotExtrinsicParams, config::Config};
+
+use crate::configuration::Configuration;
+use crate::error::AppError;
+use crate::utils::{calculate_signature, get_current_block, get_tx_counter};
+
+use self::kilt::runtime_types;
+use self::kilt::runtime_types::did::did_details::DidAuthorizedCallOperation;
+
+#[subxt::subxt(runtime_metadata_path = "./metadata.scale")]
+pub mod kilt {}
+
+pub type ProxyType = kilt::runtime_types::peregrine_runtime::ProxyType;
```

Comment:
This is still hardcoded to peregrine. I'd propose to hardcode it against spiritnet or make it configurable using a feature flag

Reply:
I added a feature: [ae31e59](https://github.com/KILTprotocol/attester/pull/1/commits/ae31e59dc840a0de8e5a3337bd5800ceb06eb29d)